managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/EntryCacheImpl.java

```diff
@@ -69,15 +69,14 @@ public String getName() {
         return ml.getName();
     }
 
-    public final static PooledByteBufAllocator ALLOCATOR = new PooledByteBufAllocator(
-            true, // preferDirect
+    public final static PooledByteBufAllocator ALLOCATOR = new PooledByteBufAllocator(true, // preferDirect
             0, // nHeapArenas,
```

Comment:
PooledByteBufAllocator.defaultNumHeapArena() instead of 0?

Reply:
No, the 0 is intentional, we only use this allocator for entry cache and we only use direct buffers